charts/gardener-dashboard/values.yaml

```diff
@@ -157,13 +157,20 @@ global:
     #   # - `${projectName}`: name of the project
     #   # - `${utcDateTimeNow}`: current date-time in utc format
     #   # - `${seedName}`: shoot's seed name
+    #   # - `${accessRestrictions}`: shoot's selected accessRestrictions
```

Comment:
As we pass an object here we should give details about it's properties, WDYT?

Reply:
added additional descriptions